src/lib/decorations/brace.typ

```diff
@@ -321,7 +321,7 @@
     // pointing upwards, so we set the origin and rotate the entire group accordingly
     let (_, start, end) = coordinate.resolve(ctx, start, end)
     set-origin(start)
-    rotate(vector.angle2(start, end) * -1)
+    rotate(vector.angle2(start, end))
```

Comment:
Can you add a comment about why it needs to be multiplied by -1?

Reply:
It does not. I've removed the `* -1`.

Reply:
*OH* my bad